contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -8,7 +9,8 @@ import "./ExecutionPrevention.sol";
 import "../interfaces/AggregatorV3Interface.sol";
 import "../interfaces/LinkTokenInterface.sol";
 import "../interfaces/KeeperCompatibleInterface.sol";
-import {Config, State} from "../interfaces/KeeperRegistryInterface.sol";
+import "./interfaces/OVM_GasPriceOracle.sol";
```

Comment:
nit. handle it in the same way as ./vendor/@arbitrum ?

Reply:
Done